### DISCUSSION TITLE
Mohist / GriefPreention / Quark - Quark chests are openable (and I dont want them to be)

### DISCUSSION THREAD

**Discussion by anu3bis** [GriefPrevention/GriefPrevention#1615] [Q&A]
I'd like to lock non-vanilla chests, same as they do for vanilla ones.
There appears to be a way to to do this:

open the config.yml file, and add the following:

  Mods:
    BlockIdsRequiringContainerTrust: [spruce_chest]

I've tried a number of formatting variations on this, ranging from using the ID# to the full name (quark_spruce_chest) to...well, a lot, but every time I start up the server again, the section is gone.  I can make trivial changes to config.yml - like changing the STICK to WOODEN_AXE.    I think I'm just formatting it wrong- any ideas?

**Comment by bigpresh** [comment]
`BlockIdsRequiringContainerTrust` doesn't appear to be a setting in GriefPrevention, it appears nowhere in the source code, and isn't documented.  Where have you found that?

Is it perhaps something for the Sponge version?  (This repo is for the Bukkit/Spigot version; we have nothing to do with the Sponge version.)

**Reply by Jikoo** [reply]
It's a legacy setting from outdated versions prior to the flattening in 1.13. https://github.com/TechFortress/GriefPrevention/wiki/Deprecated-options

**Comment by Jikoo** [comment]
Mohist needs to implement some form of container interface for GP to be able to detect these blocks. GP's handling could be better (#1142)   but it does protect generic containers pretty well, so I have to assume that Mohist is not translating modded blocks well to Bukkit objects.

**Reply by anu3bis** [reply]
Fair - following up with them:  https://github.com/MohistMC/Mohist/issues/1780

**Reply by GnomoMuitoLoco** [reply]
Existe solução?

**Comment by anu3bis** [comment]
https://dev.bukkit.org/projects/grief-prevention/pages/setup-and-configuration
Mod Support section.

"*Mods.BlockIdsRequiringAccessTrust: [1, 2, 3] *

Any block IDs you list here will require /accesstrust from the owner before
a player can interact with them (usually right click). I've heard several
mods introduce some kind of "machines" which you can activate by right
clicking them. Those would be good candidates for adding here."

**Reply by GnomoMuitoLoco** [reply]
Isso foi resolvido? estou com o mesmo problema... Mesmo protegendo conseguem abrir, meu servidor possui mods e uso Mohist
